src/test/scala/com/ubirch/webui/TestRefUtil.scala

```diff
@@ -242,7 +246,11 @@ object TestRefUtil extends LazyLogging with Matchers with Elements {
     val providerClaimedGroup = GroupFactory.getByName(Util.getProviderClaimedDevicesName(provider))
 
     // check attributes
-    deviceAttributes(Elements.ATTRIBUTES_API_GROUP_NAME).toArray shouldBe apiAttributes.head._2.toArray
+    val apiAttributeHead = deviceAttributes(Elements.ATTRIBUTES_API_GROUP_NAME).toArray.head.toString
+    implicit val formats: DefaultFormats.type = DefaultFormats
+    val json = parse(apiAttributeHead)
+    val pwd = (json \ "password").extract[String]
+    UUID.fromString(pwd) // will fail if not a uuid
```

Comment:
where is the errorhandling? Might also throw an exception, not?

Reply:
Well that's the good thing, if it throws an error, the test will fail, as expected ;). This util class is only used in tests.
However, that's true that it'd be "cleaner" to put a proper error handling, catch it in the test scenario, then fail later on with a more explicit message.

Reply:
ah non, probably I just didn't see, that it was a test class.. ;)